website/src/pages/_app.tsx

```diff
@@ -2,7 +2,24 @@ import { ReactElement } from 'react';
 import { AppProps } from 'next/app';
 import '@theguild/components/style.css';
 import '../hotfix.css';
+import localFont from 'next/font/local';
+
+const neueMontreal = localFont({
+  src: [
+    { path: '../fonts/PPNeueMontreal-Regular.woff2', weight: '400' },
+    { path: '../fonts/PPNeueMontreal-Medium.woff2', weight: '500' },
```

Comment:
what about if we move fonts as static assets in guild-components? if you need help with it ping me

Reply:
We definitely can do it, but these will still need to be relative paths. I tried to move the whole `next/font` machinery once, and turns out the way Next is handling it doesn't allow that.